src/main/resources/application.properties

```diff
@@ -27,7 +27,6 @@ hocs.basicauth=UNSET
 aws.region=eu-west-2
 aws.account.id=12345
 
-cache.user.refresh=3600
 cache.team.refresh=3600
```

Comment:
Although not related - could this be removed as well, we no longer have this team cache refresh either.

Reply:
I have made this change

Reply:
Sorry, it's gone now